src/llmq/instantsend.cpp

```diff
@@ -510,7 +510,11 @@ void CInstantSendManager::ProcessTx(const CTransaction& tx, bool fRetroactive, c
     // block after we retroactively locked all transactions.
     if (!IsInstantSendMempoolSigningEnabled() && !fRetroactive) return;
 
-    if (!TrySignInputLocks(tx, fRetroactive, Params().GetConsensus().llmqTypeInstantSend)) return;
+    if (!TrySignInputLocks(tx, fRetroactive, Params().GetConsensus().llmqTypeInstantSend)) {
+        if (!TrySignInputLocks(tx, fRetroactive, Params().GetConsensus().llmqTypeDIP24InstantSend)) {
+            return;
+        }
+    }
```

Comment:
We shouldn't do this, we should only sign for what we believe is the current tip